File: back-end/Inlog.Desafio.Backend/Inlog.Desafio.Backend.Domain/Models/Veiculo/Veiculo.cs

namespace Inlog.Desafio.Backend.Domain.Models.Veiculo{

   public class Veiculo
   {
       public long Id { get; set; }
       public string Chassi { get; set; }
       public TipoVeiculo TipoVeiculo { get; set; }
       public string Cor { get; set; }
       public string Placa{ get; set; }
       public string NumeroSerieRastreador{ get; set; }
       public Localizacao Localizacao { get; set; }
   
   }
}

File: front-end/DesafioFrontend/src/app/models/Veiculo.ts
import { Localizacao } from './Localizacao';
export interface Veiculo {
  id?: number,
  chassi: string,
  tipoVeiculo: number,
  cor: string,
  placa: string,
  numeroSerieRastreador: string,
  localizacao: Localizacao
}
